langchain4j-core/src/main/java/dev/langchain4j/store/embedding/EmbeddingStore.java

```diff
@@ -52,14 +54,39 @@ public interface EmbeddingStore<Embedded> {
      */
     List<String> addAll(List<Embedding> embeddings);
 
+    /**
+     * Adds multiple embeddings and their corresponding contents that have been embedded to the store.
+     *
+     * @param ids A list of embeddings ids associated with the added embeddings.
+     * @param embeddings A list of embeddings to be added to the store.
+     * @param embedded   A list of original contents that were embedded.
+     * @return A list of IDs associated with the added embeddings.
+     */
+    List<String> addAll(List<String> ids, List<Embedding> embeddings, List<Embedded> embedded);
```

Comment:
nit: please move it below `addAll(List<Embedding> embeddings, List<Embedded> embedded)`

Reply:
+++